go.mod

```diff
@@ -3,40 +3,38 @@ module github.com/libp2p/go-libp2p-core
 go 1.17
 
 require (
-	github.com/benbjohnson/clock v1.3.0
-	github.com/btcsuite/btcd/btcec/v2 v2.1.3
-	github.com/gogo/protobuf v1.3.1
 	github.com/ipfs/go-cid v0.2.0
-	github.com/ipfs/go-log/v2 v2.5.1
-	github.com/libp2p/go-buffer-pool v0.0.2
-	github.com/libp2p/go-flow-metrics v0.1.0
-	github.com/libp2p/go-msgio v0.0.6
-	github.com/libp2p/go-openssl v0.0.7
-	github.com/minio/sha256-simd v1.0.0
-	github.com/mr-tron/base58 v1.2.0
-	github.com/multiformats/go-multiaddr v0.4.1
-	github.com/multiformats/go-multicodec v0.4.1
-	github.com/multiformats/go-multihash v0.0.15
-	github.com/multiformats/go-varint v0.0.6
-	github.com/stretchr/testify v1.7.0
+	github.com/libp2p/go-libp2p v0.21.1-0.20220818111105-ab797a1b0417
```

Comment:
Need to update after https://github.com/libp2p/go-libp2p/pull/1688 is merged.

Reply:
What's the ordering here. Should https://github.com/libp2p/go-libp2p/pull/1688 happen first?

Reply:
Yes, this needs to reference v0.22.0.